presto-main/etc/log.properties

```diff
@@ -9,3 +9,4 @@ io.prestosql=INFO
 com.sun.jersey.guice.spi.container.GuiceComponentProviderFactory=WARN
 com.ning.http.client=WARN
 io.prestosql.server.PluginManager=DEBUG
+com.amazonaws=WARN
```

Comment:
Why do we want to add this to the development configs? Is this related to adding the JAXB dependency?

Reply:
Adding this helped to discover the warning from aws sdk, I thought it might be useful to discover similar issues in future. I can remove it if it's not useful to have it here.

Reply:
INFO is the default level, so this is decreasing the level to WARN. Do you not see warnings without this?

I was concerned that this PR was causing the SDK to log more messages at INFO level and you were thus trying to hide them.

Reply:
My bad, I somehow mistakenly thought that the warning wasn't showing up with the default config but I've now confirmed that it does. Reverted this change.